mkosi/qemu.py

```diff
@@ -219,19 +220,20 @@ def run_qemu(args: MkosiArgs, config: MkosiConfig) -> None:
     if config.output_format not in (OutputFormat.disk, OutputFormat.cpio, OutputFormat.uki):
         die(f"{config.output_format} images cannot be booted in qemu")
 
-    if config.output_format == OutputFormat.uki and config.qemu_firmware not in (QemuFirmware.auto, QemuFirmware.uefi):
-        die(f"uki images cannot be booted with the '{config.qemu_firmware}' firmware")
-
-    if config.output_format == OutputFormat.cpio and config.qemu_firmware not in (QemuFirmware.auto, QemuFirmware.linux, QemuFirmware.uefi):
-        die(f"cpio images cannot be booted with the '{config.qemu_firmware}' firmware")
+    if config.output_format in (OutputFormat.cpio, OutputFormat.uki) and\
+        config.qemu_firmware not in (QemuFirmware.auto, QemuFirmware.linux, QemuFirmware.uefi):
+        die(f"{config.output_format} images cannot be booted with the '{config.qemu_firmware}' firmware")
```

Comment:
Line continuations are hard to spot, let's use parentheses
```suggestion
    if (config.output_format in (OutputFormat.cpio, OutputFormat.uki) and
        config.qemu_firmware not in (QemuFirmware.auto, QemuFirmware.linux, QemuFirmware.uefi)):
        die(f"{config.output_format} images cannot be booted with the '{config.qemu_firmware}' firmware")
```
This way one knows and needs to look at the end of the line too, to see where the condition ends.
```suggestion
    if (
        config.output_format in (OutputFormat.cpio, OutputFormat.uki) and
        config.qemu_firmware not in (QemuFirmware.auto, QemuFirmware.linux, QemuFirmware.uefi)
    ):
        die(f"{config.output_format} images cannot be booted with the '{config.qemu_firmware}' firmware")
```
would also visually separate the body.